lib/cookie_consent_functions.php

```diff
@@ -39,6 +39,7 @@ public static function cookie_consent_backend() {
 		$button_content = rex_config::get('cookie_consent', 'button_content');
 		$link_content = rex_config::get('cookie_consent', 'link_content');
 		$link = rex_config::get('cookie_consent', 'iLink');
+		$link_target_type = rex_config::get('cookie_consent', 'link_target_type');
```

Comment:
Sollte wie in [L121-L124](https://github.com/FriendsOfREDAXO/cookie_consent/pull/20/files#diff-037ad815c25266674072ced92eb0c0d1R121)